### PR TITLE
update sample upload response

### PR DESCRIPTION
**Pull request by mindy-stivers**

### CODE REVIEW

File: content/docs/partner-uploads.md [hunk 1]
```diff
@@ -31,8 +31,8 @@ Here is an example request.
 	    "id": 1570175274,
 	    "external_id": "Lunch_Run.gpx",
 	    "error": null,
-	    "status": "Your activity is ready.",
-	    "activity_id": 1456449319
+	    "status": "Your activity is still being processed.",
+	    "activity_id": null
```
Comment (zackisaacs) [inline content/docs/partner-uploads.md]: We actually don't return the `activity_id` field at all in this case. I think it makes sense to omit it altogether.

Comment (hariniriyer) [reply content/docs/partner-uploads.md]: @zackisaacs I tried this out and i get `activity_id: null` on `POST /uploads.

Comment (mindy-stivers) [reply content/docs/partner-uploads.md]: I copied the status and activity_id directly from the response to an upload I did this afternoon. It looks like we do include it.

Comment (zackisaacs) [reply content/docs/partner-uploads.md]: Whoopsies, you are totally correct.

**Pull request merged**